src/content/guides/installation.md

```diff
@@ -44,7 +44,7 @@ Installing locally is what we recommend for most projects. This makes it easier
 }
 ```
 
-T> To run the local installation of webpack you can access its bin version as `node_modules/.bin/webpack`.
+T> To run the local installation of webpack you can access its bin version as `node_modules/.bin/webpack`. Alternatively, if you are using npm v5.2.0 or greater, you can use npx by running 'npx webpack'. This will run the locally installed binary of webpack.
```

Comment:
```suggestion
T> To run the local installation of webpack you can access its binary version as `node_modules/.bin/webpack`. Alternatively, if you are using npm v5.2.0 or greater, you can run 'npx webpack' to do it.
```

Reply:
This is shorter, and not saying twice that we are going to run local binary by it

Reply:
how was this resolved @lkounadis or argued against?

Reply:
I believe that it is a bit more elaborate as is tbh because it explains after what npx does.

Reply:
I do not agree that repeating it twice in the same context (think: hint block) is a good idea, we can wait for more opinions but i am not approving this